### DISCUSSION TITLE
Data-Science

### DISCUSSION THREAD

**Discussion by abughurab16** [EddieHubCommunity/support#804] [General]
here is the link to the 66daysofdata website 
https://www.66daysofdata.com/

**Comment by acloudpotato** [comment]
Thank you

**Comment by abughurab16** [comment]
your welcome!

**Comment by abughurab16** [comment]
Good videos to watch before jumping into The DS/ML/AI field
https://www.youtube.com/watch?v=q8cEt8gj3zY
https://www.youtube.com/watch?v=XWetgrNas-k

**Comment by abughurab16** [comment]
some great resources to get started are
https://kaggle.com/
https://deepnote.com/
https://66daysofdata.com/
